apps/ledger-live-mobile/e2e/models/onboarding/onboardingSteps.ts

```diff
@@ -43,11 +41,10 @@ export default class OnboardingSteps {
   }
 
   async chooseToExploreApp() {
-    // Fixme : Found a way to skip discover carousel without disabling synchro (animations prevent click)
-    if (isAndroid()) await device.disableSynchronization();
+    await device.disableSynchronization(); // Animations prevent click
     await tapByElement(this.exploreAppButton());
-    if (!isAndroid()) await device.disableSynchronization();
-    for (let i = 0; i < 4; i++) {
+    // Fixme : Found a way to skip discover carousel first page on iOS
```

Comment:
Does this mean iOS has 1 less page than Android?

Reply:
There is the same number of page. On iOS we have an element not clickable error when clicking on first page